packages/zoe/test/swingsetTests/zoe/test-zoe.js

```diff
@@ -298,12 +268,12 @@ const expectedBadTimerLog = [
   'aliceMoolaPurse: balance {"brand":{},"value":3}',
   'aliceSimoleanPurse: balance {"brand":{},"value":0}',
 ];
-test('zoe - bad timer', async t => {
+test.skip('zoe - bad timer', async t => {
```

Comment:
There should be an issue for this, and there should be a TODO here referring to the issue.

Reply:
There is an issue. https://github.com/Agoric/agoric-sdk/issues/1625 I will add TODOs